backend/functions/lib/adapters/gateways/storage/local-storage-gateway.ts

```diff
@@ -55,10 +61,33 @@ function getRoleMapping(): Map<string, CamsRole> {
   return roleMapping;
 }
 
+function addUstpDivisionMetaToMap(
+  map: Map<string, UstpDivisionMeta>,
+  meta: UstpDivisionMeta,
+  divisionCodes: string[],
+) {
+  divisionCodes.forEach((divisionCode) => {
+    if (map.has(divisionCode)) {
+      map.set(divisionCode, { ...map.get(divisionCode), ...meta });
+    } else {
+      map.set(divisionCode, meta);
+    }
+  });
+}
+
+function getUstpDivisionMeta(): Map<string, UstpDivisionMeta> {
+  if (!metaMapping) {
+    metaMapping = new Map<string, UstpDivisionMeta>();
+    addUstpDivisionMetaToMap(metaMapping, { isLegacy: true }, LEGACY_DIVISION_CODES);
+  }
+  return metaMapping;
+}
```

Comment:
I am not seeing what this code is doing for us. It seems that we're never adding any codes to `LEGACY_DIVISION_CODES` so line 69 has nothing to iterate over.

Reply:
I did have "070" in here, but Jon confirmed that the division had no legacy cases so I moved it to the "invalid" list in the gateway. I left the legacy logic in place because I'm pretty sure we will be adding legacy division codes in the near term.